types/src/genesis/mod.rs

```diff
@@ -2,8 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use address::{json as addr_json, Address};
+use fvm_shared::bigint::bigint_ser as bigint_json;
```

Comment:
This does not look right. If it _is_ right, please explain why.

Reply:
Fixed